Service/Helpers/AttributeHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Xml.Linq;
+
+namespace XamlMagic.Service.Helpers
+{
+	internal static class AttributeHelper
+	{
+		/// <summary>
+		/// Gets a properly configured XName object from an XAttribute.
+		/// </summary>
+		/// <param name="attribute"></param>
+		/// <returns></returns>
+		public static XName GetName(XAttribute attribute)
+		{
+			if(attribute.Value.Contains(":"))
+			{
+				string[] parts = attribute.Value.Split(':');
+
+				if (parts.Length == 2)
+				{
+					string prefix = parts[0];
+					string value = parts[1];
+
+					var ns = attribute.Parent.GetNamespaceOfPrefix(prefix);
```

Comment:
I know there are violations of this all over, so don't worry about this one, but in general, avoid abbreviations, as they are more difficult to parse to someone not as familiar with the code.